legal-api/src/legal_api/resources/v2/business/business_filings/business_documents.py

```diff
@@ -64,11 +65,15 @@ def get_documents(identifier: str, filing_id: int, legal_filing_name: str = None
 
     if identifier.startswith("T"):
         filing_model = FilingModel.get_temp_reg_filing(identifier)
-        legal_entity = LegalEntity.find_by_internal_id(filing_model.legal_entity_id)
+        business = None
+        if filing_model.legal_entity_id:
+            business = LegalEntity.find_by_internal_id(filing_model.legal_entity_id)
+        elif filing_model.alternate_name_id:
+            business = AlternateName.find_by_id(filing_model.alternate_name_id)
```

Comment:
as mentioned previously, there will be a function for this in `business_service` soon.